skll/utils/commandline/filter_features.py

```diff
@@ -12,21 +12,23 @@
 import logging
 import sys
 from pathlib import Path
+from typing import List, Optional
 
 from skll.data.readers import EXT_TO_READER, safe_float
 from skll.data.writers import EXT_TO_WRITER, ARFFWriter, CSVWriter, TSVWriter
 from skll.version import __version__
 
 
-def main(argv=None):
+def main(argv: Optional[List[str]] = None) -> None:
```

Comment:
oh, nvm. it's not argv, it's just a varname. 🤦

Reply:
We are already using `argparse` if you look at the code. The `argv` bit is needed to accommodate testing since we want to be able to pass in arguments programmatically.